golang.mk

```diff
@@ -0,0 +1,121 @@
+# This is the default Clever Golang Makefile.
```

Comment:
Can you copy in the versioned copy? https://raw.githubusercontent.com/Clever/dev-handbook/master/make/golang.mk